RedHatInsights/insights-inventory-frontend
# fix(RHINENG-8550): Fix the last seen filter not being checked

Fixes https://issues.redhat.com/browse/RHINENG-8550.

This resolves one of the issues started happening after PF5 migration: the last seen filter values were not marked as checked when user selected any of the values.

It slightly changes how we build filter configuration for the Last seen filter: the value was not marked as selected, because PrimaryToolbar (from frontend-components) expects `value` to be string, whereas we were passing an object which should have broken selection.

## How to test

- At /inventory, make sure the filter works correctly: the selected option is checked, the table is updated and URL has a filter parameter. After refresh, the filter must be applied as well (to the exception of "custom" option).
- On any other page where InventoryTable is used, test if the filter works there as well.
